File: 프로그래머스/0/181924. 수열과 구간 쿼리 3/수열과 구간 쿼리 3.cs
using System;

public class Solution {
    public int[] solution(int[] arr, int[,] queries) {
        int[] answer = arr;
        for(int i = 0 ; i < queries.GetLength(0); i++)
        {
            int x = queries[i,0];
            int y = queries[i,1];
            int temp = answer[x];
            answer[x] = answer[y];
            answer[y] = temp;
        }
        return answer;
    }
}

File: 프로그래머스/0/181924. 수열과 구간 쿼리 3/수열과 구간 쿼리 3.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, vector<vector<int>> queries) {
    vector<int> answer = arr;
    for(int i = 0 ; i < queries.size(); i++)
    {
        int x = queries[i][0];
        int y = queries[i][1];
        
        int temp = answer[x];
        answer[x] = answer[y];
        answer[y] = temp;
    }
    return answer;
}